Kernel_23/doc/Kernel_23/CGAL/Weighted_point_2.h

```diff
@@ -47,11 +47,8 @@ class Weighted_point_2
 
   /*!
   introduces a weighted point from point `p` and weight `0`.
-
-\warning The `explicit` keyword is used to not have accidental implicit conversions
-         between Point_2 and Weighted_point_2.
   */
-  explicit Weighted_point_2(const Point_2<Kernel>& p);
```

Comment:
Why do you remove that `explicit` keyword? Is not it the goal of this work: temove the implicit conversions?